CHANGELOG.md

```diff
@@ -4,6 +4,24 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # Latest
+* Update Razor to 9.0.0-preview.24327.6 (PR: [#7273](https://github.com/dotnet/vscode-csharp/pull/7273))
```

Comment:
@dibarbet want to make sure this is in the right spot since I believe a snap has happened since I opened this.

Reply:
@ryzngard I just merged the changelog updates back to main, you will have to resolve the conflicts.  Then this change should go into a new Latest section, and you can rename the previous latest section 2.37.x (don't know the final number yet).

Reply:
Perfect. Just merged